vulture/utils.py

```diff
@@ -54,12 +54,15 @@ def format_path(path):
 
 
 def get_decorator_name(decorator):
-    if isinstance(decorator, ast.Call):
-        decorator = decorator.func
     parts = []
-    while isinstance(decorator, ast.Attribute):
-        parts.append(decorator.attr)
-        decorator = decorator.value
+    while True:
+        if isinstance(decorator, ast.Call):
+            decorator = decorator.func
+        while isinstance(decorator, ast.Attribute):
+            parts.append(decorator.attr)
+            decorator = decorator.value
+        if not isinstance(decorator, ast.Call):
+            break
```

Comment:
in the regression test this will return something in the lines of `@hist.labels.time` I'm not completely sure if this is what is wanted, but atleast now in that specific case it's not failing 🤔

Reply:
The other option would be to return calls with `args`/`kwargs` preserved (like `@hist.labels("labelvalue").time`).

For context, the reason why `get_decorator_method` exists is for us to be able to support the `--ignore-decorators` flag. Preserving `args`/`kwargs` here would make it possible for users to ignore for particular arguments/kwargs only (e.g. `--ignore-decorators "@hist.labels("labelA").time"`), but I'm not sure it's possible to do that statically. Besides, even if it is, it would require significant changes to the code and I don't think it worth the maintenance effort. If a user wants to ignore for one value of `labelValue`, they probably want to do so for other values too.